x/fee/client/cli/query.go

```diff
@@ -96,7 +105,13 @@ func GetNonFungibleTokenFeeSetting(cdc *codec.Codec) *cobra.Command {
 
 			symbol := args[0]
 
-			res, _, err := cliCtx.QueryWithData(fmt.Sprintf("custom/%s/%s/%s", "fee", fee.QueryNonFungibleTokenFeeSetting, symbol), nil)
+			res, _, err := cliCtx.QueryWithData(fmt.Sprintf("custom/%s/%s/%s", "nonFungible", nonfungible.QueryTokenData, symbol), nil)
+			if err != nil {
+				fmt.Printf("Could not get non-fungible token fee setting: %s\n", err)
```

Comment:
@githubckgoh1439 fix the error message.

Reply:
Fixed the message using 'non-fungible token not exist'.